File: ClaimRuler/Lib/SchedulerNet/Samples/Scheduler.MVC3/Controllers/MultiSchedulerController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using DHTMLX.Scheduler;
namespace SchedulerTest.Controllers
{
    /// <summary>
    /// you can create multiple scheduler on page
    /// </summary>
    public class MultiSchedulerController : Controller
    {
        //
        // GET: /MultiScheduler/
        public class mod
        {
            public DHXScheduler sh1 { get; set; }
            public DHXScheduler sh2 { get; set; }
        }
        public ActionResult Index()
        {
            //each scheduler must have unique name
            var scheduler = new DHXScheduler("sched1");
  
            scheduler.InitialView = scheduler.Views[scheduler.Views.Count - 1].Name;
            var scheduler2 = new DHXScheduler("sched2");
   
            return View(new mod() { sh1 = scheduler, sh2 = scheduler2 });
        }

    }
}

File: ClaimRuler/Lib/SchedulerNet/Samples/Scheduler.MVC3.VB/Controllers/MultiSchedulerController.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports DHTMLX.Scheduler
Namespace SchedulerTest.Controllers
    ''' <summary>
    ''' you can create multiple scheduler on page
    ''' </summary>
    Public Class MultiSchedulerController
        Inherits Controller
        '
        ' GET: /MultiScheduler/
        Public Class [mod]

            Public sh1 As DHXScheduler
            Public sh2 As DHXScheduler
        End Class
        Public Function Index() As ActionResult
            'each scheduler must have unique name
            Dim scheduler = New DHXScheduler("sched1")

            Dim scheduler2 = New DHXScheduler("sched2")

            Return View(New [mod]() With { _
             .sh1 = scheduler, _
             .sh2 = scheduler2 _
            })
        End Function

    End Class
End Namespace
